### PR TITLE
docs(pt-br): Update pt-br/index.mdx

### PR DESCRIPTION
**Pull request by Exerea**
## Summary
Update pt-br homepage.

Related: https://github.com/biomejs/biome/issues/1958

## Test Plan
Run pnpm start locally and check/compare the translated homepage on the dev server with the default one.

### CODE REVIEW

File: website/src/content/docs/pt-br/index.mdx [hunk 1]
```diff
@@ -4,11 +4,15 @@ head:
   - tag: title
     content: Biome, conjunto de ferramentas da web
 template: splash
-description: Formate, verifique erros e muito mais em uma fração de segundo.
+description: Formato, fiapos e muito mais em uma fração de segundo.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:7]: @Exerea This change doesn't look good for me (as a native speaker)

File: website/src/content/docs/pt-br/index.mdx [hunk 2]
```diff
@@ -190,7 +188,7 @@ import LinterExample from "@/components/linter/example.md";
   <h2 class="head">Comunidade</h2>
   <div class="community-connect">
     <div class="community-description">
-      Desenvolvido pela nossa incrível comunidade
+      Alimentado por nossos colaboradores de código aberto
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:191]: @Exerea I think in this case "Mantido" or "Desenvolvido" is more appropriate here

File: website/src/content/docs/pt-br/index.mdx (hunk 1 above)
Comment (victor-teles) [reply website/src/content/docs/pt-br/index.mdx:7]: Also, you need to keep the same sentence on the site hero section

File: website/src/content/docs/pt-br/index.mdx [hunk 3]
```diff
@@ -4,11 +4,15 @@ head:
...
+description: Formato, fiapos e muito mais em uma fração de segundo.
 editUrl: false
 next: false
+banner:
+  content: |
+    Considere fazer uma doação para a instituição de caridade <a href="https://www.enduringhearts.org/" target="_blank">Enduring Hearts</a>
+
 hero:
-  title: Um conjunto de ferramentas para seu projeto web
+  title: Cadeia de ferramentas integrada para o desenvolvimento de projetos da Web.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/index.mdx:15]: @Exerea You need to keep the same sentence on the site hero section

**Review by victor-teles — CHANGES_REQUESTED** [review]

**Comment by Exerea** [comment]
@victor-teles  Thank you for your feedback! I have updated it.

**Review by victor-teles — APPROVED** [review]

**Pull request merged**